docs/NSR.md

```diff
@@ -104,13 +104,13 @@ f.	Maintain, archive, and retain logs in accordance with disclosed business prac
 
 Certification Authorities and Delegated Third Parties SHALL:
 
-a.	Implement intrusion detection and prevention controls under the control of CA or Delegated Third Party Trusted Roles to protect Certificate Systems against common network and system threats;
+a.	Except for offline Root CA systems, implement intrusion detection and prevention controls under the control of CA or Delegated Third Party Trusted Roles to protect Certificate Systems against common network and system threats;
 
 b.	Document and follow a vulnerability correction process that addresses the identification, review, response, and remediation of vulnerabilities; 
 
-c.	Undergo or perform a Vulnerability Scan (i) within one (1) week of receiving a request from the CA/Browser Forum, (ii) after any system or network changes that the CA determines are significant, and (iii) at least every three (3) months, on public and private IP addresses identified by the CA or Delegated Third Party as the CA’s or Delegated Third Party’s Certificate Systems;
+c.	Undergo or perform a Vulnerability Scan (i) within one (1) after any system or network changes that the CA determines are significant, and (ii) at least every three (3) months, on public and private IP addresses identified by the CA or Delegated Third Party as the CA’s or Delegated Third Party’s Certificate Systems;
 
-d.	Undergo a Penetration Test on the CA’s and each Delegated Third Party’s Certificate Systems on at least an annual basis and after infrastructure or application upgrades or modifications that the CA determines are significant;
+d.	Undergo a Penetration Test on all non-offline Root CA’s and each Delegated Third Party’s Certificate Systems on at least an annual basis and after infrastructure or application upgrades or modifications that the CA determines are significant;
```

Comment:
"Undergo a Penetration Test on all network connected Root CA’s and each Delegated Third Party’s Certificate Systems on at least an annual basis and after infrastructure or application upgrades or modifications that the CA determines are significant;"

Reply:
Do you really want this to apply to air-gapped systems?